server/server.py

```diff
@@ -978,8 +374,7 @@ def generate_and_apply_city_layout(self, city_size):
 if __name__ == "__main__":
```

Comment:
all these changes make it so we can't instance two servers at once. need to future proof to let server owners run multiple servers from one run of the software.

Reply:
Can you elaborate on this point? Based off of the current structure of the code, I don't think it will work that way now anyways:
* World directory is hardcoded to ./worlds/default and there is no room for differentiating server instance in this hierarchy.
* Accounts: `./accounts/<ident>` same thing.
* No thread pooling; the server will only run on the main virtual thread, and there is no way to interact with it through the console.
* There is only one socket, and all existing logic assumes that there will only be one socket (player identification is purely based off of the strings they pass, not with their ip:port combination)

Generally speaking, if you want the abilitity to run two instances of the server through one run of the software, then the best way would be to design the server as an object that can be instantiated on a new thread by a small harness that would manage server threads. (Example of one approach to do this: https://docs.python.org/2/library/multiprocessing.html#using-a-pool-of-workers)

As I mentioned above, in order to do this with any strategy, there cannot be shared memory between instances.  If this is your intended goal moving forward, then I think my change is a necessary first step to realizing it.